src/trackers/BT.py

```
@@ -26,6 +26,13 @@
 
 
 class BT:
+    tracker = "BT"
+    banned_groups: list[str] = []
```

Ruff (0.15.20)

[warning] Mutable default value for class attribute

(RUF012)